### ISSUE TITLE
Incorrect indentation with .sass files

### ISSUE THREAD

**Issue by saminton** [gazugafan/vscode-indent-on-paste#8]
Title pretty much says it all. I prefer to use the older .sass format instead of .scss. I guess the extension can't calculate the proper indentation due to the lack of brackets.

The possibility to disable the extension for certain languages would also do. It works like a charm in php files.

**Comment by gazugafan** [comment]
Yeah, unfortunately the generic indentation algorithm needs some sort of closing block to work off of. So stuff like SASS and Python won't really work with this. They need a proper language server extension that understands the language.

I'll see about getting an extension exclusion feature added soon, though!
